test/src/tests/account.test.js

```diff
@@ -8,19 +7,18 @@ import {
   vIcon,
 } from '../vuetify-xpaths';
 
-
-beforeAll(async () => {
-  // Set the default action timeout to something greater than 500ms
-  expectPuppeteer.setDefaultOptions({ timeout: 10000 });
-  await page.goto(CLIENT_URL);
-});
-
 describe('account', () => {
   const username = `user${uniqueId()}`;
-  const email = `${username}@kitware.com`;
+  const email = `${username}@dandi.test`;
   const password = 'password'; // Top secret
 
+  beforeAll(async () => {
+    await page.goto(CLIENT_URL);
+  });
+
   it('register', async () => {
+    expect.assertions(9);
```

Comment:
My understanding is that this statement is for testing expects in callbacks. The `await`s effectively de-asynchronize these tests, so `expect.assertions` doesn't provide any value that I can see. Does the jest eslint plugin require it/generate it automatically? I would rather disable that for this project.

Reply:
Agreed.